lib/Protocol.hpp

```diff
@@ -84,6 +84,7 @@ struct proxy_wifi_bss
 #pragma warning(disable : 4200)
 struct proxy_wifi_scan_response
 {
+    uint8_t scan_complete;
```

Comment:
this isn't a bool?

Reply:
Functionally, it's a bool. But for types that are shared with the guest, I use fixed size types to ensure the layout is the one I want.